lib/react/jsx/processor.rb

```diff
@@ -0,0 +1,17 @@
+module React
+  module JSX
+    class Processor
+      def self.instance
+        @instance ||= new
+      end
+      
+      def self.call(input)
+        instance.call(input)
+      end
+
+      def call(input)
+        @output ||= JSX::transform(input[:data])
```

Comment:
Hmmm as you can see in the specs, this `@output` is getting assigned in the first use, but then never being reassigned. This causes all inputs to have the same output! 

Is there a downside to simply 
```
class Processor 
  def self.call(input)
    JSX::transform(input[:data])
  end 
end
```

?